lib/node_modules/@stdlib/blas/base/wasm/scasum/test/test.ndarray.js

```diff
@@ -0,0 +1,157 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Complex64Array = require( '@stdlib/array/complex64' );
+var scasum = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is an object', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof scasum, 'object', 'main export is an object' );
+	t.end();
+});
+
+tape( 'the `ndarray` method has an arity of 4', function test( t ) {
+	t.strictEqual( scasum.ndarray.length, 4, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the `main` method computes the sum of the absolute values of the real and imaginary components of a single-precision complex floating-point vector', function test( t ) {
```

Comment:
```suggestion
tape( 'the `ndarray` method computes the sum of the absolute values of the real and imaginary components of a single-precision complex floating-point vector', function test( t ) {
```

This change applies throughout this file.